integ/failure-server/run.sh

```diff
@@ -14,6 +14,14 @@ set -eo pipefail
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 TUF_REFERENCE_REPO="${DIR}/../../tough/tests/data/tuf-reference-impl"
 
+# if we are under cygwin , set a windows style path
+# (docker for windows requires windows style paths to work ,it doesn't understand cygwin-style paths)
+if [[ "$OSTYPE" == "cygwin" ]]; then
```

Comment:
even`MinGW` is famous for windows. However, i think there is no generic way to detect windows [OS] (https://stackoverflow.com/a/8597411)
@webern  should we consider to support MinGW ?

Reply:
I think if it fixes the script in the contributor's workflow then I'm happy with it. Mostly we will need to lock things down with CI (#349) and document what tools are needed for the developer workflow on Windows.

Reply:
I don't think it would. We don't need it as of now.